api/src/main/java/org/apache/cloudstack/api/command/user/snapshot/CopySnapshotCmd.java

```diff
@@ -84,6 +85,15 @@ public class CopySnapshotCmd extends BaseAsyncCmd implements UserCmd {
                     "Do not specify destzoneid and destzoneids together, however one of them is required.")
     protected List<Long> destZoneIds;
 
+    @Parameter(name = ApiConstants.STORAGE_ID_LIST,
+            type=CommandType.LIST,
+            collectionType = CommandType.UUID,
+            entityType = StoragePoolResponse.class,
+            required = false,
+            description = "A comma-separated list of IDs of the storage pools in other zones in which the snapshot will be made available. " +
```

Comment:
any case to copy to different pools in the same zone where snapshot doesn't exists?

Reply:
@sureshanaparti, there isn't an option to copy the snapshot to a different pool in the same zone.